src/main.js

```diff
@@ -11,13 +11,13 @@
  */
 
 // External depenencies
-require('fullcalendar');
-var $ = require('jquery');
-var timekit = require('timekit-sdk');
-var moment = require('moment');
+var $             = require('jquery');
+var fullcalendar  = window.fullcalendar = require('fullcalendar');
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['fullcalendar' is defined but never used.](https://www.codacy.com/app/laander/booking-js/file/1386052570/issues/source?bid=2969108&fileBranchId=2980436#l15)